bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -169,6 +170,12 @@
      */
     String PN_DESIGN_DYNAMIC_MEDIA_ENABLED = "enableDmFeatures";
 
+    /**
+     * Name of the configuration policy property that the {@link AdaptiveImageServlet} will use for resizing images that don't provide a width information in their request.
+     * @since com.adobe.cq.wcm.core.components.models 12.22.0
+     */
+    String PN_RESIZE_WIDTH = "resizeWidth";
```

Comment:
We generally prefix policy properties with `PN_DESIGN_<name>` and use `PN_<name>` for simple/resource properties.